boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -166,7 +167,13 @@
 	vbp = <14>;
 	backlight-gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
 	data-bus-width = "24-bit";
-	pixel-format = "rgb-565";
+	pixel-format = <LCDIF_PIXEL_FORMAT_BGR_565>;
+	/*
+	 * Pixel clock is given by the following formula:
+	 * (height + vsync + vfp + vbp) *
+	 * (width + hsync + hfp + hbp) * frame rate
+	 */
+	pixel-clock = <62346240>;
```

Comment:
if this property can be computed from other information already available, why add it?